doc/gemma_online.md

```diff
@@ -0,0 +1,34 @@
+===GEMMA Online API pagina
```

Comment:
Denk dat je een kopje wilt maken?

```suggestion
## GEMMA Online API pagina
```

Reply:
Ah, ja, zo dus. In de media wiki moet dat dan weer met "=" ...
<h1> en <h2>  enzo is veel handiger ;-)